scripts/ingests/ingest_BYW_2024.py

```diff
@@ -0,0 +1,35 @@
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+from astropy.io.ascii import ascii
+import astropy.units as u
+from astropy.coordinates import Angle
+
+SAVE_DB = True  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+
+# live google sheet
+link = "https://docs.google.com/spreadsheets/d/1JFa8F4Ngzp3qAW8NOBurkz4bMKo9zXYeF6N1vMtqDZs/pub?output=csv"
+
+columns = ['source', 'ra', 'dec', 'epoch', 'equinox', 'shortname', 'reference', 'other_ref', 'comments']
+byw_table = ascii.read(link, format='ecsv', data_start=2, data_end=90, header_start=1, names=columns, guess=False,
+                           fast_reader=False, delimiter=',')
+
+data_columns = ['Source', 'RA', 'Dec', 'Epoch', 'Equinox', 'Shortname', 'Reference', 'Other_ref', 'Comments']  # columns with wanted data values
+
+
+def ingest_source(db):
```

Comment:
Careful. A function already exists with this name. You imported it from utils. This function should be named something different.